### PR TITLE
clarify unclear content and/or terms, add new content (definitions, c…

### LINKED ISSUE #1215: [Edit] HTML: Forms
### Type of Edit (select all that apply)

Clarify unclear content and/or terms, Add new content (definitions, codeblocks, etc.)

### Description (optional)

We'd like the following edits to be made for [HTML: Forms](https://www.codecademy.com/resources/docs/html/forms) entry, in the `html/concepts/forms/forms.md` file:

- **Bold** the first use of the word "forms".
- On line # 19, change "tag" to "element".
- Add a "Syntax" section that offers a minimal \```pseudo code block of what an HTML form would typically look like.
  - This should include the `<form>`, `<label>`, and `<input>` elements.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by farah-qistina**
### Description

HTML Forms, forms.md
Bold the first use of the word "forms".
On line # 19, change "tag" to "element".
Add a "Syntax" section that offers a minimal ```pseudo code block of what an HTML form would typically look like.
Closes #1215 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #`<issue number>` in the "Description" above).

### CODE REVIEW

File: content/html/concepts/forms/forms.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Forms'
-Description: 'In HTML, forms are used for gathering and sending user input to an external source for processing.'
+Description: 'In HTML, __forms__ are used for gathering and sending user input to an external source for processing.'
```
Comment (SSwiniarski) [inline content/html/concepts/forms/forms.md:3]:
```suggestion
Description: 'In HTML, **forms** are used for gathering and sending user input to an external source for processing.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@farah-qistina I did a first review and only have one comment.

**Review by SSwiniarski — APPROVED** [review]
@farah-qistina Looks good.

File: content/html/concepts/forms/forms.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Forms'
-Description: 'In HTML, forms are used for gathering and sending user input to an external source for processing.'
+Description: 'In HTML, **forms** are used for gathering and sending user input to an external source for processing.'
```
Comment (yangc95) [inline content/html/concepts/forms/forms.md:3]:
```suggestion
Description: 'In HTML, forms are used for gathering and sending user input to an external source for processing.'
```

File: content/html/concepts/forms/forms.md [hunk 3]
```diff
@@ -16,12 +16,12 @@ CatalogContent:
...
 
-In HTML, forms are used for gathering and sending user input to an external source (like a [server](https://www.codecademy.com/resources/docs/general/server)) for processing. This is done with the [`<form>`](https://www.codecademy.com/resources/docs/html/elements/form) tag, which contains various labels and [inputs](https://www.codecademy.com/resources/docs/html/elements/input).
+In HTML, forms are used for gathering and sending user input to an external source (like a [server](https://www.codecademy.com/resources/docs/general/server)) for processing. This is done with the [`<form>`](https://www.codecademy.com/resources/docs/html/elements/form) element, which contains various [labels](https://www.codecademy.com/resources/docs/html/elements/label) and [inputs](https://www.codecademy.com/resources/docs/html/elements/input).
 
-## Example
-
-The following example
+## Syntax
 
+The following example shows what an HTML form would typically look like with a `<form>` element that features `<input>` elements and its corresponding `<label>` elements:
```
Comment (yangc95) [inline content/html/concepts/forms/forms.md:23]:
```suggestion
The following example shows what an HTML form that features `<input>` elements:
```

I actually don't see any labels in the example. Did you mean to add them to the example?

**Review by yangc95 — COMMENTED** [review]
@farah-qistina I conducted a quick review! Please tag me once you've resolved all the comments/suggestions or for discussion on something 🙂

**Comment by farah-qistina** [comment]
@yangc95, I wrongly assumed that there were labels in the example, sorry about that! I've edited the file according to your reviews and committed the changes, please have a look :)

**Review by yangc95 — COMMENTED** [review]
Thanks for working on this edit @farah-qistina!

I mainly have one more rework request for this entry which is to create a new code block underneath syntax to only display a the form element rather than an example. We can then add back the original example header. It's okay if this entry follows term entries as this will most likely be moved into another concept.

Thanks for your patience and let me know if you need direct assistance ☺️

**Comment by SSwiniarski** [comment]
@farah-qistina, It's been two weeks, do you need assistance doing @yangc95's suggested edit?

**Comment by farah-qistina** [comment]
I've committed my changes @SSwiniarski @yangc95

File: content/html/concepts/forms/forms.md [hunk 4]
```diff
@@ -16,12 +16,20 @@ CatalogContent:
...
-In HTML, forms are used for gathering and sending user input to an external source (like a [server](https://www.codecademy.com/resources/docs/general/server)) for processing. This is done with the [`<form>`](https://www.codecademy.com/resources/docs/html/elements/form) tag, which contains various labels and [inputs](https://www.codecademy.com/resources/docs/html/elements/input).
+In HTML, **forms** are used for gathering and sending user input to an external source (like a [server](https://www.codecademy.com/resources/docs/general/server)) for processing. This is done with the [`<form>`](https://www.codecademy.com/resources/docs/html/elements/form) element, which contains various [labels](https://www.codecademy.com/resources/docs/html/elements/label) and [inputs](https://www.codecademy.com/resources/docs/html/elements/input).
 
-## Example
+## Syntax
 
-The following example
+```html
+<form>
+<!-- form elements -->
```
Comment (yangc95) [inline content/html/concepts/forms/forms.md:25]:
```suggestion
  <!-- Form elements -->
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks for these edits @farah-qistina ☺️

live link: https://www.codecademy.com/resources/docs/html/forms

Feel free to take up open [Issues](https://github.com/Codecademy/docs/issues), if any.

**Comment by farah-qistina** [comment]
@yangc95 thx for letting me work on this, I learned a lot!

**Comment by yangc95** [comment]
@farah-qistina yeah, follow these steps:

> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin
>
> If the steps above do not work, it may be due to an email or local configuration issue.
>
> In addition, we may reach out in the future if any adjustments to these entries are required.

**Pull request merged**